### PR TITLE
Replaced prettier with standard, created command to lint js and sass

### PR DESCRIPTION
**Pull request by mayavera**
> ⚠️ blocked by #43 _Clean preact usage_

# Changelog
- removed old lint deps
- added deps for `eslint`, `eslint-standard`, and `stylelint`
- removed old lint configs
- added configs for `eslint` that extends standard presets
- added config for `stylelint` that extends standard preset

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,33 +60,33 @@ seatmapApi.highlightSection('section-id');
...
+| ticketGroups       |          | `[]`                                                                                                                                                                                                                                               | An array of ticket groups to be used for section pricing. This API was designed for you to directly pass into the client library the response from the [Ticket Evolution `/v9/ticket_groups` endpoint](https://ticketevolution.atlassian.net/wiki/spaces/API/pages/9469962/Ticket+Groups+Index). Expects each ticket group to adhere to the [TicketGroup interface](#interfaces).                                                                                               |
 
 # Public API
 
-| Name | Input | Output | Notes
-| - | - | - | - |
-| updateTicketGroups | `TicketGroup[]` | `void` | Changes the collection of ticket groups in the map used to calculate available sections and section prices. Useful if you have a feature for filtering ticket groups and you want the map to update.
```
Comment (josephspens) [inline README.md]: Are we removing the description to this method?

File: src/ticketmap/colors.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+  getAllPaths().forEach(element => element.setAttribute('data-unavailable-color', element.getAttribute('fill')))
 }
 
 export const fillUnavailableColors = () => {
-  getAllPaths().forEach(element => element.setAttribute('fill', element.getAttribute('data-unavailable-color')));
+  getAllPaths().forEach(element => element.setAttribute('fill', element.getAttribute('data-unavailable-color')))
 }
 
 export const getAllPaths = (id) =>
-  Array.from(document.querySelectorAll(`[data-section-id${!!id ? `="${id}"` : ''}]`))
```
Comment (josephspens) [inline src/ticketmap/colors.js]: Interesting, was this the formatter or was it a manual change?

File: src/ticketmap/index.js [hunk 3]
```diff
@@ -372,88 +370,88 @@ export default class TicketMap extends Component<*, State> {
...
+      this.toggleSectionSelect(section, !this.state.selectedSections.includes(section))
     }
   }
 
-  render(): ?React$Element<any> {
+  render (): ?React$Element<any> {
     return (
       <div
-        ref={element => this.rootRef = element}
-        onMouseOver={this.onMouseOver.bind(this)}
```
Comment (josephspens) [inline src/ticketmap/index.js]: Why are we removing bindings?

File: src/ticketmap/tooltip.js [hunk 4]
```diff
@@ -1,58 +1,71 @@
-import CSSTransition from 'react-transition-group/CSSTransition';
-
-import { mainContainerStyle } from './styles';
+import PropTypes from 'prop-types'
```
Comment (josephspens) [inline src/ticketmap/tooltip.js]: Why are we adding prop types?

Comment (mayavera) [reply src/ticketmap/tooltip.js]: `standard-react` requires prop-type definitions on all components

File: README.md (hunk 1 above)
Comment (mayavera) [reply README.md]: Nice catch! My formatter must have eaten it.

File: src/ticketmap/colors.js (hunk 2 above)
Comment (mayavera) [reply src/ticketmap/colors.js]: This was the work of `eslint --fix`

File: src/ticketmap/index.js (hunk 3 above)
Comment (mayavera) [reply src/ticketmap/index.js]: We aren't?

Comment (mayavera) [reply src/ticketmap/index.js]: It's already bound via the `onMouseOver = () => {}`. We would need to bind it if it was `onMouseOver() {}`.

File: src/ticketmap/tooltip.js (hunk 4 above)
Comment (mayavera) [reply src/ticketmap/tooltip.js]: We should strive to stick to standard as much as possible. It should make the project easier to read and modify by other developers and people integrating the library into their site. However, I think forgoing the prop types, for the time being, is the right choice (given we're moving to TypeScript in the immediate future)

**Pull request merged**